ui/app/AppLayouts/Profile/Sections/Contacts/ContactList.qml

```diff
@@ -15,6 +15,10 @@ ListView {
     property string lowerCaseSearchString: searchString.toLowerCase()
     property string contactToRemove: ""
 
+    property Component profilePopupComponent: ProfilePopup {
+        id: profilePopup
```

Comment:
You need to add a `destroy()` call `onClosed` since you create a new object instance each time.